Requirements catalog

vlade44 opened the pull request.

OHUSAR committed:
[specification] add WIP requirements catalog

vlade44 committed:
Minor text fixes

vlade44 committed:
added output specification, added index for variables

OHUSAR reviewed (changes requested):
**Firstly**

Open office aj WPS Writer majú s tým wordom problémy, takže možno niečo niekde nevidím tak, ako to je a preto možno tam tá formátovacia chyba nie je - _sorry about that_.

---

**Nesplnené požiadavky Petra Gergela**
- namiesto SUM pouzit symbol sumy
  -- s týmto nesúhlasím, pretože je to poľa mňa skôr pseudokód a môže to tak ostať (IMHO)
- strucne vysvetlit co je premenna p-type (**toto musime nejako zistit :D**)

---

**Doplniť referencie:**
    - odkazy na použité články a stránky (?)

---

**Formátovacie nedostatky:**

3.3 je nejaké celé rozhádzané, ale neviem čo konkrétne je tom zle. Predpokladám, že to len nemá mať formátovanie ako nadpis.

V popise vstupných parametrov by som dal položky ako napr.
"Param.strings" boldom, kvôli prehladnosti. Tak isto v popise
X a Y transformácií by som dal hodnoty pred zátvorkou boldom.
Ako napr. (Xout, atd.)

---

**Preklepy**

Úplne prvá veta, odstrániť _medzeru pred bodkou._

Druhá strana úplne dole - Menno1@Meno2@...@MenoN - _Odstrániť hyperlink._

Posledná veta v celom dokumente - Budeme implementovať Doppler, Lorentz, Voigt, Hartmann-Tran - _bodka na konci vety._

Pred otváracou zátvorkou má byť medzera - viac krát.

**Celkovo by si to zaslúžilo prehnať cez nejaký fungujúci spellcheck.**

vlade44 committed:
Fixed major issues

- pridane referencie
- upravene formatovanie (bold, undrelined, ..) kvoli prehladnosti
- pridany popis k premennym [*strucne vysvetlit co je premenna p-type]
- opravene preklepy (hopefully)
- SUM som nechal tak ako to je, podla mna to je fajn format

filipKov committed:
edit requirements specification
- former specification file
+ moved to separate folder
+ formatting
+ header
+ authors, instructor names
+ more export formats

filipKov committed:
remove former spec file
- former software requirements specification file ( forgot to include it to former commit )

OHUSAR committed:
Merge pull request #9 from TIS2016/requirementsSpec

Give it a little Filip Kovac touch :+1:

OHUSAR commented:
**Peter Cermak**

---

Trochu som este doplnil chybajuce veci...

Ide o to ze ci:
- aspon +- rozumiete tomu co idete programovat:)
- potrebujete este presnejsie specifikacie do tohto dokumentu (ohladom struktury co pride z LB) alebo treba tam aj popisat tie profily ciar (tej tabulke chyba caption a citacia)

Jeho pripomienkovana verzia:

[softwareRequirementsSpecification_03.docx.zip](https://github.com/TIS2016/Analyza-spektroskopickych-dat/files/529429/softwareRequirementsSpecification_03.docx.zip)

---

vlade44 committed:
If approved, this will be final version of our srs

- Added changes specified by P.Cermak
- Did spellcheck and fixed typos

vlade44 committed:
Updates in formulas

- SUM prepisane pomozou vzorcou
- Pridal som vzorce pre jednotlive profily

vlade44 committed:
Add files via upload

vlade44 committed:
Final version of srs upload (.doc .pdf)

OHUSAR commented:
![schvalenie](https://cloud.githubusercontent.com/assets/14111857/20539754/810d7f26-b0f6-11e6-8026-a03987eebbcc.png)
